submissions/Neskazhuk/popup/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&display=swap" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <nav class="nav">
+            <ul class="nav-list">
+                <li class="nav-item"><a class="link" href="#" tabindex="1">Hello</a></li>
+                <li class="nav-item"><a class="link" href="#" tabindex="2">World</a></li>
+                <li class="nav-item popup">
+                    <input id="popup-input-id" class="popup-input" type="checkbox" tabindex="3">
+                    <label for="popup-input-id" class="popup-btn">
+                        <img class="popup-icon" src="./icons/popup-button.png" alt="popup icon">
+                    </label>
+                    <div class="popup-container" tabindex="-1">
+                        <input id="more-input" class="more-input" type="checkbox" tabindex="16">
+                        <label for="more-input" class="more-btn">
+                        </label>
+                        <ul class="popup-list">
+                            <li class="popup-item" tabindex="4">
```

Comment:
Adding an attribute to list items makes them focused items, but you still have to interact with interactive items (links), so move the attribute to the link.
Also, no need to increment `tabindex` for sibling element.  You can use the same value for all links inside `li`.

Reply:
Before, you said there is no need to nest tabindex inside sibling elements, so I moved them to parent <li> elements. Now we need to put them back?   Why we cannot just leave them somewhere and move forward?

Reply:
When a blind user with a screen reader will use your site your markup will be focused on `li` items and if he wants to click on the chosen link he must do extra moves to find the link before he can click on it. You must provide focus on interactive elements.

Reply:
Moved tabindex to the links. Used the same value for most of them(as you asked)